src/seedu/addressbook/AddressBook.java

```diff
@@ -596,6 +603,23 @@ private static String executeListAllPersonsInAddressBook() {
         showToUser(toBeDisplayed);
         return getMessageForPersonsDisplayedSummary(toBeDisplayed);
     }
+    
+    /**
+     * Displays all persons in the address book to the user; in alphabetical order.
+     * 
+     * @return feedback display message for the operation result
+     */
+    private static String executeListAllPersonsInAddressBookInAlphabetOrder() {
+    	ArrayList<String[]> toBeDisplayed = getAllPersonsInAddressBook();
+    	Comparator<String[]> comparator = new Comparator<String[]>() {
+    		public int compare(String[] p1, String[] p2) {
+    			return getNameFromPerson(p1).compareTo(getNameFromPerson(p2));
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.